src/main/java/edu/cornell/kfs/fp/document/RecurringDisbursementVoucherDocument.java

```diff
@@ -0,0 +1,86 @@
+package edu.cornell.kfs.fp.document;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.sys.businessobject.GeneralLedgerPendingEntry;
+import org.kuali.kfs.sys.businessobject.GeneralLedgerPendingEntrySequenceHelper;
+import org.kuali.kfs.sys.businessobject.GeneralLedgerPendingEntrySourceDetail;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.coreservice.framework.parameter.ParameterConstants.COMPONENT;
+import org.kuali.rice.coreservice.framework.parameter.ParameterConstants.NAMESPACE;
+import org.kuali.rice.kew.api.exception.WorkflowException;
+import org.kuali.rice.kew.framework.postprocessor.DocumentRouteStatusChange;
+
+import edu.cornell.kfs.fp.businessobject.RecurringDisbursementVoucherDetail;
+import edu.cornell.kfs.fp.service.RecurringDisbursementVoucherDocumentService;
+
+@NAMESPACE(namespace = KFSConstants.CoreModuleNamespaces.FINANCIAL)
+@COMPONENT(component = "RecurringDisbursementVoucher")
+public class RecurringDisbursementVoucherDocument extends CuDisbursementVoucherDocument {
```

Comment:
Are we gaining value from extending CuDisbursementVoucherDocument (and ultimately AccountingDocumentBase and *PostingDocumentBase) here? If we could avoid that (by extending FinancialSystemTransactionalDocumentBase maybe) we could avoid having to override the GLPE methods with noops.